docs/basic.rst

```diff
@@ -49,13 +48,13 @@ Call it in your base template, for example:
     {{ bootstrap.load_js() }}
     </body>
 
-You can pass ``version`` to pin the Bootstrap 4 version you want to use. Default to load files from CDN, set ``BOOTSTRAP_SERVE_LOCAL``
-to ``True`` to use built-in local files. However, it's recommended to manage Bootstrap resources by yourself.
+You can pass ``version`` to pin the Bootstrap 4 version you want to use. It defaults to load files from CDN. Set ``BOOTSTRAP_SERVE_LOCAL``
+to ``True`` to use built-in local files. However, it's recommended to let Bootstrap manage its resources by itself.
```

Comment:
What I want to express here is it's recommended to let the user write `<href></href>` and `<script></script>` codes to include Bootstrap resources instead of using `bootstrap.load_css()` and `bootstrap.load_js()`.

Reply:
Ok. So if you want to let people manage the resources themselves, maybe phrase it as:
"...to use local files. That will give the advantages of full control and is the recommended method for advanced users".
If this is really the preferred way , but not the standard setting, you should add a section to the advanced install or something.
I am really ok if you change the pull request or commit a change afterwards.